kubernetes/loculus/values.yaml

```diff
@@ -1721,7 +1721,8 @@ defaultOrganisms:
           noInput: true
           generateIndex: true
           autocomplete: true
-          ingest: segment
+          preprocessing:
```

Comment:
oh this actually doesn't work - I wonder if its because `segment` is a custom field - definitely the previous fix worked